### PR TITLE
 Populating Players Profile

### PR DESCRIPTION
**Pull request by sydbui**
Populated Players Profile

Completed Features
- pulled data from players table and populated the player profile with the information for the Overview, Engagement, Academic Performace, and Physical Wellness tabs. 
- created component for text fields used in the player's tabs 
- created component for score box in the Academic Performace tab
- restructured files for player
- created dynamic routing to get players profile

To-Do
- reference Users table to get the name of a player
- more refactoring 

## PR Checklist

Does your branch (check if true):
[X] work when you run `yarn build`?
[X] successfully run `yarn:db-migrate up` without yielding any errors?
[X] successfully run `yarn prisma introspect` without yielding any errors?
[X] successfully run `yarn dev` and support all changes that your branch intends to perform?

## Related PRs

N/A

## Migrations

N/A

## Screenshots

Screen Shot 2020-10-30 at 6 49 16 PM
Screen Shot 2020-10-30 at 6 49 36 PM
Screen Shot 2020-10-30 at 6 49 48 PM
Screen Shot 2020-10-30 at 6 50 03 PM

To check this PR make sure to seed your database, run yarn dev, and then search http://localhost:3000/player/[whatever player number you want (has to be greater than 12)]

CC: @erinysong

### CODE REVIEW

File: pages/player/PlayerNavBar.tsx [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+          </div>
+          <div className="mb-6 text-lg font-bold">Student Bio</div>
+          <div className="m-5">
+            <TextLayout title="About Me" content={player.bio} />
+            <TextLayout title="Hobbies" content={player.bio} />
+            <TextLayout title="Favorite Subject" content={player.bio} />
+            <TextLayout title="Most Difficult Subject" content={player.bio} />
+            <TextLayout title="Siblings" content={player.bio} />
+            <TextLayout title="Parents" content={player.bio} />
+            <TextLayout title="Favorite Fifa Game" content={player.bio} />
```
Comment (erinysong) [inline pages/player/PlayerNavBar.tsx:26]: apparently favorite fifa game was a meme so it looks like we don't have to include it anymore :")

File: pages/player/[id].tsx [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+    async function getPlayer(): Promise<void> {
+      try {
+        const response = await fetch(
+          `http://localhost:3000/api/players/read/${ID}`,
+          {
+            method: "GET",
+            headers: { "content-type": "application/json" },
+          }
+        );
+        const player1: Player & {
```
Comment (erinysong) [inline pages/player/[id].tsx:66]: let's rename this to playerdata for clarity

**Review by erinysong — CHANGES_REQUESTED** [review]
looks great! added a couple of comments for minor nits but otherwise it should be good to go - super happy with how this turned out!

File: pages/player/[id].tsx (hunk 2 above)
Comment (sydbui) [reply pages/player/[id].tsx]: made all changes!

**Comment by shannonmhong** [comment]
This looks amazing! Idk what else to say haha

**Pull request merged**